src/auro-menu.js

```diff
@@ -103,7 +103,7 @@ class AuroMenu extends LitElement {
    * @returns {Boolean} Returns true if the option is interactive.
    */
   optionInteractive(option) {
-    return !option.hasAttribute('disabled') && !option.hasAttribute('static');
+    return !option.hasAttribute('hidden') && !option.hasAttribute('disabled') && !option.hasAttribute('static');
```

Comment:
I am unsure how to actually test this? From the bug description in the issue, I am not seeing a difference between this branch and the demo in the doc site. 

To test, I took `main` of combobox and linked to the working menu branch. 

Also, there should be a new test, or an updated test for this feature.

Reply:
The demo page includes an example labeled "Menu with hidden options". On the doc site you can see it move 'active' to the hidden option as you move through the list with the arrow keys. In localhost it correctly skips the hidden option.

We didn't previously have a test that covered skipping disabled items either. I added one to cover both hidden and disabled.